enterprise/app/org/org_members.tsx

```diff
@@ -250,7 +250,8 @@ export default class OrgMembersComponent extends React.Component<OrgMembersProps
         <div className="affected-users-list">
           {selectedMembers.map((member) => (
             <div className={`affected-users-list-item ${this.isLoggedInUser(member) ? "flagged-self-user" : ""}`}>
-              {member?.user?.email || member?.user?.name?.full} {iconFromAccountType(member.user?.accountType)}
+              {member?.user?.email || member?.user?.name?.full || member?.user?.username}{" "}
```

Comment:
This implementation looks inconsistent with the `accountLabel` implementation in `org_join_requests.tsx`. Could we extract an `accountName` function from that `accountLabel` function (could be added to `app/auth/user.ts`) and call it both here and in `accountLabel`?